contrail_api_cli/client.py

```diff
@@ -72,6 +73,10 @@ def register_argparse_arguments(self, parser):
                                     type=str,
                                     default=os.environ.get('CONTRAIL_API_PROTOCOL', 'http'),
                                     help="protocol used (default=%(default)s)")
+        contrail_group.add_argument('--baseuri',
```

Comment:
I'd prefer to have a --base-uri option
and CONTRAIL_API_BASE_URI env variable
and base_uri variable in the code

Reply:
ok, will update it